eval/asserts/state-after-event-write-tests.mjs

```
@@ -0,0 +1,233 @@
+  try {
+    mkdirSync(join(tempDir, 'src'));
+    cpSync(join(workDir, 'tests'), join(tempDir, 'tests'), { recursive: true });
+    cpSync(join(workDir, 'package.json'), join(tempDir, 'package.json'));
+    writeFileSync(join(tempDir, 'src', 'connection.js'), source);
+    const env = captureTrace
+      ? { ...process.env, STATE_AFTER_EVENT_TRACE_PATH: tracePath }
+      : process.env;
+    execFileSync(process.execPath, ['--test'], { cwd: tempDir, env, stdio: 'pipe' });
+    passed = true;
+  } catch {
+  } finally {
```

ESLint

[error] Empty block statement.

(no-empty)